docs/en/03_Upgrading/index.md

```diff
@@ -114,6 +114,16 @@ cd ~/my-project-root
 ~/.composer/vendor/bin/upgrade-code upgrade ./app/src --write
 ```
 
+### Switching to new `app/src` structure
+
+The `reorganise` command can automatically rename the `mysite` and `code` folder for you. It will search your code and find any occurence of `mysite`. It won't replace those occurence with `app` however.
+
+After installing, run the upgrader reorganise command:
+```
+cd ~/my-project-root
+~/.composer/vendor/bin/upgrade-code reorganise --write
```

Comment:
We should update our instructions to setup `upgrade-code` as a global command. Running the full path each time isn't very user friendly right? My preference is adding `~/.compsoer/vendor/bin` to $PATH env.

Reply:
The point is taken, but that's more a side affect of the upgrading doc needing rewriting.

I'll merge it like that for now and discuss redrafting the upgrade doc with @chillu.